src/main/java/com/github/javafaker/Relationships.java

```diff
@@ -0,0 +1,59 @@
+package com.github.javafaker;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Relationships {
+    private final Faker faker;
+
+    protected Relationships(final Faker faker) {
+        this.faker = faker;
+    }
+
+    public String direct() {
+        return faker.resolve("relationship.familial.direct");
+    }
+    
+    public String extended() {
+        return faker.resolve("relationship.familial.extended");
+    }
+    
+    public String in_law() {
+        return faker.resolve("relationship.in_law");
+    }
+
+    public String spouse() {
+        return faker.resolve("relationship.spouse");
+    }
+    
+    public String parent() {
+        return faker.resolve("relationship.parent");
+    }
+    
+    public String sibling() {
+        return faker.resolve("relationship.sibling");
+    }
+    
+    public String any() {
+        String[] relationships = new String[] {"direct", "extended", "in_law", "spouse", 

```

Comment:
couldn't you get a list of declaredMethods and exclude any? instead defining your own list of method names

Reply:
Fixed.